src/plugins/dashboard/public/application/application.ts

```diff
@@ -45,9 +46,10 @@ import {
   AppMountParameters,
 } from 'opensearch-dashboards/public';
 import { UsageCollectionSetup } from 'src/plugins/usage_collection/public';
+import { DashboardListSources } from 'src/plugins/dashboard/public/types';
 import { Storage } from '../../../opensearch_dashboards_utils/public';
 // @ts-ignore
-import { initDashboardApp } from './legacy_app';
+import { DashboardListItem, DashboardListProviderFn, initDashboardApp } from './legacy_app';
```

Comment:
I see this import in the diff but `DashboardListItem` && `DashboardListProviderFn` dont seem to be used anywhere in this file